examples/analog_hamiltonian_simulation/04_Maximum_Independent_Sets_with_Analog_Hamiltonian_Simulation.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<p>To clarify the definition,</p><p>"... any subset $V_{is}$ of $V$ such that every pair of vertices in $V_{is}$ are not connected by ..."</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>